pkg/api/dataproxy.go

```diff
@@ -59,3 +62,9 @@ func ensureProxyPathTrailingSlash(originalPath, proxyPath string) string {
 
 	return proxyPath
 }
+
+var proxyPathRegexp = regexp.MustCompile("^\\/api\\/datasources\\/proxy\\/[\\d]+\\/?")
+
+func getProxyPath(originalRawPath string) string {
+	return proxyPathRegexp.ReplaceAllString(originalRawPath, "")
```

Comment:
Can you just move the call to `ensureProxyPathTrailingSlash` here, is there any need to require the caller to take care of this?

Reply:
@domasx2 pointed out that ensureProxyPathTrailingSlash is no longer needed. Please have a look again